src/definitions/modules/modal.js

```diff
@@ -160,6 +163,7 @@ $.fn.modal = function(parameters) {
               subtree   : true
             });
             module.debug('Setting up mutation observer', observer);
+            module.observer = observer;
```

Comment:
As mentioned above, `observer` is already declared as private module variable and set by https://github.com/Semantic-Org/Semantic-UI/blob/0a42dcdcb400d6fc275b04d6cb052a48efd68536/src/definitions/modules/modal.js#L157, so this line should not be necessary